20240508-S-sidecar-endpoint-tls.md

```diff
@@ -182,7 +184,7 @@ What changes or actions are required to make this proposal complete?
 | dns:[2001:db8:1f70::999:de8:7648:6e8]:5000?abc=[]            |                                                            |                                   |      |       | Error: query parameters are not supported for gRPC endpoints: 'abc=[]'             |
 | dns://myauthority:53/[2001:db8:1f70::999:de8:7648:6e8]       | dns://myauthority:53/[2001:db8:1f70::999:de8:7648:6e8]:443 | [2001:db8:1f70::999:de8:7648:6e8] | 443  | FALSE |                                                                                    |
 | dns:[2001:db8:1f70::999:de8:7648:6e8]                        | dns:[2001:db8:1f70::999:de8:7648:6e8]:443 | [2001:db8:1f70::999:de8:7648:6e8] | 443  | FALSE  |                                                                             |
-| https://[2001:db8:1f70::999:de8:7648:6e8]                    | dns:[2001:db8:1f70::999:de8:7648:6e8]:443 | [2001:db8:1f70::999:de8:7648:6e8] | 443  | TRUE   |                                                                             |
+| https://[2001:db8:1f70::999:de8:7648:6e8]                    | dns:[2001:db8:1f70::999:de8:7648:6e8]:80  | [2001:db8:1f70::999:de8:7648:6e8] | 80   | TRUE   |                                                                             |
```

Comment:
Is this correct?

Reply:
Yes, "https" determines it to be a secure endpoint, not the arbitrary port. I put 80 to convey the message.